radix-engine/src/system/system_modules/costing/fee_table.rs

```diff
@@ -54,17 +81,321 @@ impl FeeTable {
+                    ("NonFungibleResourceManager", "mint_uuid") => 324056,
+                    ("NonFungibleResourceManager", "non_fungible_exists") => 174706,
+                    ("NonFungibleResourceManager", "update_non_fungible_data") => 269194,
+                    ("NonFungibleVault", "create_proof_of_all") => 236221,
+                    ("NonFungibleVault", "create_proof_of_amount") => 226422,
+                    ("NonFungibleVault", "create_proof_of_non_fungibles") => 229425,
+                    ("NonFungibleVault", "get_amount") => 92070,
+                    ("NonFungibleVault", "get_non_fungible_local_ids") => 93095,
+                    ("NonFungibleVault", "lock_non_fungibles") => 156268,
+                    ("NonFungibleVault", "put") => 158008,
+                    ("NonFungibleVault", "recall") => 267720,
+                    ("NonFungibleVault", "take") => 228152,
+                    ("NonFungibleVault", "take_non_fungibles") => 228294,
+                    ("NonFungibleVault", "unlock_non_fungibles") => 156532,
+                    ("Package", "PackageRoyalty_claim_royalty") => 542170,
+                    ("Package", "PackageRoyalty_set_royalty_config") => 256237,
+                    ("Package", "publish_wasm") => 521324,
+                    ("Proof", "Proof_drop") => 225030,
+                    ("Proof", "Proof_get_amount") => 82525,
+                    ("Proof", "Proof_get_non_fungible_local_ids") => 84021,
+                    ("Proof", "Proof_get_resource_address") => 79495,
+                    ("Proof", "clone") => 232409,
+                    ("Radiswap", "instantiate_pool") => 11222983,
+                    ("Radiswap", "swap") => 3381443,
+                    ("TransactionProcessor", "run") => 2040533,
+                    ("Validator", "claim_xrd") => 1019930,
+                    ("Validator", "register") => 341788,
+                    ("Validator", "stake") => 1292740,
+                    ("Validator", "unregister") => 299565,
+                    ("Validator", "unstake") => 1620979,
+                    ("Validator", "update_accept_delegated_stake") => 313021,
+                    ("Validator", "update_key") => 386889,
+                    ("Worktop", "Worktop_drain") => 80356,
+                    ("Worktop", "Worktop_drop") => 76247,
+                    ("Worktop", "Worktop_put") => 244598,
+                    ("Worktop", "Worktop_take") => 320777,
+                    ("Worktop", "Worktop_take_all") => 68469,
+                    ("Worktop", "Worktop_take_non_fungibles") => 174567,
+                    ("Package", "publish_native") => (input_size * 13 + 10908) >> 2, // calculated using linear regression on gathered data
+                    ("Package", "publish_wasm_advanced") => input_size * 25 + 218153, // calculated using linear regression on gathered data
+                    _ => 454121, // average of above values
```

Comment:
At the moment, native blueprints and wasm blueprints are calling `consume_cost_units` directly by themselves. 

We need a way to consolidate the two mechanisms.